libraries/AdaptiveExpressions/parser/ExpressionAntlrLexer.g4

```diff
@@ -83,6 +83,8 @@ mode STRING_INTERPOLATION_MODE;
 
 STRING_INTERPOLATION_END : '`' {ignoreWS = true;} -> type(STRING_INTERPOLATION_START), popMode;
 
+EOL : '\r'? '\n';
+
 TEMPLATE : '$' '{' (STRING | OBJECT_DEFINITION | ~[\r\n{}'"`])+ '}';
 
 ESCAPE_CHARACTER : '\\' ~[\r\n]?;
```

Comment:
How about let the "TEXT_CONTENT" contains "\r?\n" ?

Reply:
good idea. changed to TEXT_CONTENT